### ISSUE TITLE
Problems for search macros/variables at-point for verilog projects development

### ISSUE THREAD

**Issue by guoqinglei** [redguardtoo/emacs.d#981]
The emacs.d from redguardtoo is the config I have been using for over three months. 
One of the reasons I like this is the fast start speed and proved high programming efficiency.
Recently, I want to use this config to develop some verilog projects. However, this config is not suitable in some cases. For example,  when I press C-] at the point of a Macro, i.e., `DEF_MACRO, it reported "No tag is found. Grep "`DEF_MACRO" at directory: /bin/bash: -c: line 0: unexpected EOF while looking for matching '`'. I wonder what I could do to fix it.

Another similar feature I am eagerly to want  is that I usually need to search the variable/word at point . I know the command counsel-etags-find-tag-at-point can be used, but It only shows some definitions of this variable/word. Actually, I want to know all the occcurences of the variable/word in the buffer, just like the way of swiper or occur command works. Is there a command like swiper-at-point or occur-at-point ?

**Comment by guoqinglei** [comment]
I have found some command:

isearch-forward-symbol-at-point

**Comment by guoqinglei** [comment]
I think the problem may still exist for me. Actually I still prefer to improve the function of counsel-etags-find-tag-at-point.
For string " `MACRO " in the verilog code, the

**Comment by redguardtoo** [comment]
You can customise `counsel-etags-find-tag-name-function`.

**Comment by redguardtoo** [comment]
Yes. The tag name at point is found here. It's used as a keyword for searching the TAGS file.
